index.js

```diff
@@ -61,7 +62,9 @@ try {
     throw({message: 'ASANA PAT Not Found!'});
   }
   if (TASK_COMMENT) {
-    taskComment = `${TASK_COMMENT} ${PULL_REQUEST.html_url}`;
+    taskComment = TASK_COMMENT;
+    if (INCLUDE_REVIEW_STATUS) taskComment += ` ${gitub.event.review.status}`;
```

Comment:
This can only be helpful if we fire the action on either `pull_request_review` or `pull_request_review_comment` otherwise it'll always be false. 
And in case of the above mentioned events all the other operations will be triggered aswell.

Also if you want to add this kindly add the inputs in `action.yml` too.